Demo/Demo/Helpers/OrderRequestHelpers.swift

```diff
@@ -10,10 +10,7 @@ enum OrderRequestHelpers {
             intent: .authorize,
             purchaseUnits: [
                 PayPalCheckout.PurchaseUnit(
```

Comment:
We want to add a ticket to our backlog to remove the Demo app's use of NXO for creating an order, as this is not what a merchant should do.